workspace/src/common/launch/art_localization_launch/launch/art_localization.launch.py

```diff
@@ -66,10 +69,12 @@ def generate_launch_description():
                 ("~/input/gyroscope", GetLaunchArgument("art_localization/input/gyroscope")),
                 ("~/input/accelerometer", GetLaunchArgument("art_localization/input/accelerometer")),
                 ("~/output/vehicle_state", GetLaunchArgument("art_localization/output/vehicle_state")),
+                ("~/input/vehicle_inputs", GetLaunchArgument("art_localization/input/vehicle_inputs")),
+                ("~/input/groundTruth", GetLaunchArgument("input/groundTruth")),
         ],
         parameters=[
-            {"vis": GetLaunchArgument("vis")},
-            {"use_sim_time": GetLaunchArgument("use_sim_time")}
+            {"use_sim_time": GetLaunchArgument("use_sim_time")},
+            {"SE_mode": GetLaunchArgument("SE_mode")}
```

Comment:
Might wanna have a more informative variable name than SE_mode. Not sure what that means without looking at the code

Reply:
Do you have any suggestions? Also I'm assuming it's not possible to have a finite set of options for what the user can input here - correct? It would be nice to limit the user to the available options (GT, EKF, PF).